engine/src/main/java/org/terasology/engine/module/StandardModuleExtension.java

```diff
@@ -47,17 +59,34 @@ public String getKey() {
     }
 
     public static boolean isServerSideOnly(Module module) {
-        Boolean serverSideOnly = module.getMetadata().getExtension(SERVER_SIDE_ONLY.getKey(), Boolean.class);
-        return serverSideOnly != null && serverSideOnly;
+        return getBooleanExtension(module, SERVER_SIDE_ONLY);
     }
 
     public static boolean isGameplayModule(Module module) {
-        Boolean isGameplay = module.getMetadata().getExtension(IS_GAMEPLAY.getKey(), Boolean.class);
-        return isGameplay != null && isGameplay;
+        return getBooleanExtension(module, IS_GAMEPLAY);
     }
 
     public static SimpleUri getDefaultWorldGenerator(Module module) {
         String ext = module.getMetadata().getExtension(DEFAULT_WORLD_GENERATOR.getKey(), String.class);
         return ext != null ? new SimpleUri(ext) : null;
     }
+
+    private static boolean getBooleanExtension(Module module, StandardModuleExtension ext) {
```

Comment:
IntelliJ + Checkstyle gave me angry red warnings on this file due to super duper evil tab characters instead of spaces. Gasp!